packages/ui/components/scheduler/components/verticalLines/index.tsx

```diff
@@ -0,0 +1,19 @@
+import dayjs from "@calcom/dayjs";
+
+export const VeritcalLines = ({ days }: { days: dayjs.Dayjs[] }) => {
+  return (
+    <div
+      className="col-start-1 col-end-2 row-start-1 grid auto-cols-auto grid-rows-1 divide-x divide-gray-300
+       sm:pr-8">
+      {days.map((_, i) => (
+        <div
+          key={`Key_vertical_${i}`}
+          className="row-span-full"
+          style={{
+            gridColumnStart: i + 1,
+          }}
+        />
+      ))}
+    </div>
+  );
+};
```

Comment:
Handles the vertical lines to separate each day. 

In future this might be handled elsewhere and be handled on one of the many grids we have with a simple divide x but that approach required some messing around with zIndex